bindings/rust/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// Fizzy: A fast WebAssembly interpreter
```

Comment:
Still surprises me that naming a file `lib.rs` is considered good practice.

Reply:
It is not "good practice", but required. The name is determined by the one in `Cargo.toml` and `lib.rs` is the default entry point.